doc/en/multitest.rst

```diff
@@ -1269,22 +1269,24 @@ Also keep in mind that testplan will take a little bit of effort to monitor exec
 Xfail
 -----
 
-You can mark testsuite/testcase that you expect to fail to testplan can deal with them accordingly and present as pass in the test report, while keeping the testsuites/testcase `green`.
+You can mark testsuite/testcase that you expect to fail testplan can deal with them accordingly and present as pass in the test report, while keeping the testsuites/testcase `green`.
```

Comment:
This sentence doesn't read very well to me. Also I think it is misleading - if an xfail test fails it will be marked as xfail (orange) not pass (green)?

How about something like:

Testcases and testsuites that you expect to fail can be marked with the `@xfail` decorator. These failures will be visible in the test report, highlighted in orange. Expected failures will not cause the testplan as a whole to be considered a failure.